cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -90,6 +90,7 @@ var fServiceName = flag.String("service-name", "telegraf", "service name (window
 var fServiceDisplayName = flag.String("service-display-name", "Telegraf Data Collector Service", "service display name (windows only)")
 var fRunAsConsole = flag.Bool("console", false, "run as console application (windows only)")
 var fSetEnv = flag.String("setenv", "", "set an env in the configuration file in the format of KEY=VALUE")
+var fStartUpErrorFile = flag.String("errorFile", "", "file to touch if agent can't start")
```

Comment:
`startup-error-file` to match format of other arguments and to be clear about the purpose.